data/hai/hai missions.txt

```diff
@@ -1557,3 +1557,57 @@ mission "Strider notification"
 			scene `thumbnail/hai pond strider`
 			`Upon landing on <planet>, you receive a message from Osen. "Hello Captain <last>! I hope this message finds you well. I would like to once again thank you for your help on the Pond Strider, and would like to inform you that we have finished implementing the hull repair technology that you have provided us into the Pond Strider design and have now begun producing the ship on a mass scale. You'll also be interested to find that we have developed new weaponry for the Pond Strider's drone, the Flea. I won't spoil the surprise for you though. Check the shipyard on Hai-home the next time you stop by to take a look at it."`
 				decline
+
+
+
+mission "Hai Translator Ban Timer"
+	invisible
+	landing
+	source
+	to offer
+		has "event: hai-human resolution announced"
+		has "Wanderers Conversation: offered"
+	on offer
+		event "Hai Translator Ban Lifted" 20 30
+		fail
+
+event "Hai Translator Ban Lifted"
+
+
+
+mission "Hai Translator Ban Lifted"
+	invisible
+	landing
+	source
+		government Hai
+	to offer
+		has "event: Hai Translator Ban Lifted"
+	on offer
+		conversation
+			`After landing, you see several humans and Hai talking to one another, each in their native language. They keep looking at a device or data pad, which seems to be translating. Some translate on-screen, while others are speaking. A Hai in an official-looking uniform comes up to you.`
+			`	"Excuse me, would you like a hai-Human translator?"`
```

Comment:
```suggestion
			`	"Excuse me, would you like a Hai-Human translator?"`
```

Reply:
Actually, I believe that should be "Hai-human"